R/target_sda.R

```diff
@@ -166,12 +166,23 @@ target_sda <- function(data,
     return(empty_target_sda_output())
   }
 
-  corporate_economy <- calculate_market_average(ald_by_sector)
+  relevant_ald_by_sector <- ald_by_sector %>%
+    filter(sector %in% unique(data$sector))
+
```

Comment:
do you need a test and warning here if `nrow(relevant_ald_by_sector) == 0L` as with  `relevant_scenario` below?

Reply:
So this is actually already tested and warned for (albeit it looks a bit different), at L164-167. Note since there is a join happening there, we can just check directly the joined dataset.